src/event.js

```diff
@@ -626,8 +631,9 @@ jQuery.event = {
 };
 
 jQuery.removeEvent = function( elem, type, handle ) {
+	// This "if" is needed for plain objects
```

Comment:
Is this a difference between a comment about the whole function and a comment targeting a specific instruction inside the function?